packages/playwright/src/matchers/expect.ts

```diff
@@ -289,8 +289,9 @@ class ExpectMetaInfoProxyHandler implements ProxyHandler<any> {
 
       const step = testInfo._addStep(stepInfo);
 
-      const reportStepError = (jestError: ExpectError) => {
-        const error = new ExpectError(jestError, customMessage, stackFrames);
+      const reportStepError = (_error: unknown) => {
```

Comment:
style: we only use `_leadingUnderscore` for private fields, not for local variable.

Reply:
done in e4b1dc95dc5155dfbad915358cda5b09f89d4c7d